ci-operator/step-registry/cucushift/installer/check/capability/cucushift-installer-check-capability-commands.sh

```diff
@@ -139,9 +139,9 @@ esac
 if [[ "${ADDITIONAL_ENABLED_CAPABILITIES}" != "" ]]; then
     enabled_capability_set="${enabled_capability_set} ${ADDITIONAL_ENABLED_CAPABILITIES}"
 else
+    declare -A additional_caps_from_config_array=()
```

Comment:
Here I think it should be `-a` instead of `-A`

Reply:
after changing `-a`, issue get fixed.
step `cucushift-installer-check-capability` is passed this time in [job](https://gcsweb-qe-private-deck-ci.apps.ci.l2s4.p1.openshiftapps.com/gcs/qe-private-deck/pr-logs/pull/openshift_release/43181/rehearse-43181-periodic-ci-openshift-openshift-tests-private-release-4.13-amd64-nightly-aws-ipi-baselinecaps-v413-f28-destructive/1701390740417941504/artifacts/aws-ipi-baselinecaps-v413-f28-destructive/cucushift-installer-check-capability/build-log.txt)